pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -157,6 +159,8 @@ func (og *operationGenerator) GenerateVolumesAreAttachedFunc(
 
 			for spec, check := range attached {
 				if !check {
+					// TODO: I think the reconciler needs to block _all_ concurrent operations - at least on the node or all volumes
```

Comment:
Don't want to block on those operations to avoid delay. Mark it as detached might not accurate at a point, but reconciler will correct it in the next round.